src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,44 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class AddCarController extends HttpServlet {
+    private static final String PAGE_PATH = "/WEB-INF/views/car/add.jsp";
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService;
+    private ManufacturerService manufacturerService;
```

Comment:
let's initialize those fields right here instead of doing it in the init() method.
and don't forget to remove that method after